config/vendor_libraries.cmake

```diff
@@ -91,8 +100,9 @@ macro( setupLAPACKLibrariesUnix )
       endif()
 
       if( BLAS_FOUND )
-        set( LAPACK_FOUND TRUE )
-        set( lapack_FOUND ON )
+        set( LAPACK_FOUND TRUE CACHE BOOL "lapack (MKL) found?")
+        set( lapack_FOUND TRUE CACHE BOOL "lapack (MKL) found?")
```

Comment:
I'll add that to be consistent.  We don't use OpenBLAS on any LANL systems, so I didn't review this section carefully.